t/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.5"
+services:
+  alpha1:
+    image: dgraph/dgraph:local
+    container_name: alpha1
+    working_dir: /alpha1
+    ports:
+    - 8180:8180
+    - 9180:9180
+    volumes:
+    - type: bind
+      source: ./
+      target: /data
+      read_only: true
+    command: dgraph alpha -o 100 --my=alpha1:7180 --zero=zero1:5180 --logtostderr
+      -v=2 --raft "idx=1; group=1" --security whitelist=0.0.0.0/0 --acl secret-file=/data/acl_secret;access-ttl=3s
+  alpha2:
+    image: dgraph/dgraph:local
+    container_name: alpha2
+    working_dir: /alpha2
+    ports:
+    - 8182:8182
+    - 9182:9182
+    volumes:
+    - type: bind
+      source: ./
+      target: /data
+      read_only: true
+    command: dgraph alpha -o 102 --my=alpha2:7182 --zero=zero1:5180 --logtostderr
+      -v=2 --raft "idx=2; group=1" --security whitelist=0.0.0.0/0 --acl secret-file=/data/acl_secret;access-ttl=3s
+  alpha3:
+    image: dgraph/dgraph:local
+    container_name: alpha3
+    working_dir: /alpha3
+    ports:
+    - 8183:8183
+    - 9183:9183
+    volumes:
+    - type: bind
+      source: ./
+      target: /data
+      read_only: true
+    command: dgraph alpha -o 103 --my=alpha3:7183 --zero=zero1:5180 --logtostderr
+      -v=2 --raft "idx=3; group=1" --security whitelist=0.0.0.0/0 --acl secret-file=/data/acl_secret;access-ttl=3s
+  zero1:
+    image: dgraph/dgraph:local
+    container_name: zero1
+    working_dir: /zero1
+    ports:
+    - 5180:5180
+    - 6180:6180
+    command: dgraph zero -o 100 --raft='idx=1' --my=zero1:5180 --replicas=3
+      --logtostderr -v=2 --bindall
```

Comment:
do we want to make this a HA on Zero's as well?

Reply:
@mangalaman93 what is your suggestion?

Reply:
either way is fine. We are just doing some sanity checks, we have tests that use dgo client in dgraph repo as well.